Home_work/Знакомство с языком Python/Семинар_4/Task_1.py

```diff
@@ -0,0 +1,7 @@
+# Вычислить число c заданной точностью d
+# Пример:
+# при d = 0.001, π = 3.142 10^(-1) ≤ d ≤10^(-10)
+
+from decimal import Decimal
+
+print(round(float(input('Введите число: ')), int(Decimal(input('Введите точность: ')).as_tuple().exponent*(-1))))
```

Comment:
```suggestion
print(Decimal(input('Введите число: ')).quantize(Decimal(input('Введите точность: '))))
```
Ни к чему такие усложнения и в одну строку решать совсем не обязательно.